pkg/trace/api/api.go

```diff
@@ -106,20 +106,10 @@ func (r *HTTPReceiver) buildMux() *http.ServeMux {
 	mux := http.NewServeMux()
 
 	r.attachDebugHandlers(mux)
-
-	mux.HandleFunc("/spans", r.handleWithVersion(v01, r.handleTraces))
-	mux.HandleFunc("/services", r.handleWithVersion(v01, r.handleServices))
-	mux.HandleFunc("/v0.1/spans", r.handleWithVersion(v01, r.handleTraces))
-	mux.HandleFunc("/v0.1/services", r.handleWithVersion(v01, r.handleServices))
-	mux.HandleFunc("/v0.2/traces", r.handleWithVersion(v02, r.handleTraces))
-	mux.HandleFunc("/v0.2/services", r.handleWithVersion(v02, r.handleServices))
-	mux.HandleFunc("/v0.3/traces", r.handleWithVersion(v03, r.handleTraces))
-	mux.HandleFunc("/v0.3/services", r.handleWithVersion(v03, r.handleServices))
-	mux.HandleFunc("/v0.4/traces", r.handleWithVersion(v04, r.handleTraces))
-	mux.HandleFunc("/v0.4/services", r.handleWithVersion(v04, r.handleServices))
-	mux.HandleFunc("/v0.5/traces", r.handleWithVersion(v05, r.handleTraces))
-	mux.HandleFunc("/v0.5/stats", r.handleStats)
-	mux.Handle("/profiling/v1/input", r.profileProxyHandler())
+	for _, e := range endpoints {
+		mux.Handle(e.Name, e.Handler(r))
+	}
+	mux.HandleFunc("/hello", r.makeHelloHandler())
```

Comment:
is it a standard practice to name a discovery endpoint `hello`? Just by the name, I would guess it's a test endpoint.
Maybe `info`, `help` or `list` ?

Reply:
🤷‍♂️ Hello sounds fine to me, but I can change it to anything else.

Reply:
I find `/hello` a little surprising to be honest. Can we change it to `/info` please?

Reply:
Sure, no problem 🙂